docs/recipes.rst

```diff
@@ -2,8 +2,8 @@ Recipes
 =======
 
 .. nbgallery::
-   :caption: Recipes
    :name: recipe-gallery
+   :caption: 
```

Comment:
I think you might just have to live with this (or come up with a pithier caption). The field is "required", and I don't know how to supply a blank value -- an inline comment and text substitution both seem to fail here. Whitespace is trimmed, then sphinx thinks there's no value, and the gallery doesn't show up at all!

Reply:
okie, it was just an attempt to see if the duplicated "Recipes" from title+main part healed